### ISSUE TITLE
namecheap.com is not compatible any more

### ISSUE THREAD

**Issue by Stopi** [beemdevelopment/Aegis#1142]
### Version

2.1.3

### Source

F-Droid

### Vault encryption

Yes (with biometric unlock)

### Device

Nubia Reg Magic 5G

### Android version

10

### ROM

Resurrection Remix

### Steps to reproduce

Go to namecheap and try to use the OTP
Or if haven't got already, try to activate 2FA using TOTP.

### What do you expect to happen?

I expect the given code to be accepted by namecheap, just like it was before.

### What happens instead?

Namecheap complains that the OTP code is either not valid or expired.

I don't know exactly when this stopped to work, probably few weeks.
They say I should use Google Authenticator or Authy.
I believe the algorithm is the same, not sure about that.
It use to work and there's even a namecheap icon present in the app.

**Comment by krisu5** [comment]
These kinds of issues usually happen when your phone's clock is incorrect. [Try syncing your phone's clock](https://support.google.com/googleplay/thread/55723112?hl=en&msgid=55728968).

**Comment by Stopi** [comment]
Hi Kristian, thanks for your answer.

I clearly understand the OTP is generating according to current time.
And of course my phone is properly configured, both for time & timezone.

Note that namecheap is the only service refusing the OTP from aegis.
Every other services are working (google, facebook, paypal, proton, binance, stripe, twilio, etc.).
If the issue was related to my phone clock, then those other services would fail too, but they don't.

Have you tried namecheap yourself? is it working for you?

**Comment by michaelschattgen** [comment]
Hi Stopi, thanks for your report. We get a lot of similiar reports where syncing the phone's clock actually solves the issue, it isn't always clear what our users know about the TOTP algorithm.

Regarding the issue you're having with Namecheap, I currently have no issue with Namecheap and my OTP code seems to work just fine. Are you sure your account hasn't been compromised?

**Comment by Stopi** [comment]
Hi Mickael,

My apologies, I did check that my phone was indeed configured to auto-sync its clock with the network before answer previous message from Kristian.
However, when unchecking then checking back the "Use network-provided time", I saw the time changed (2 mins shift).
That seems to have solved the problem with namecheap.

So in the end I think namecheap is more sensitive to that issue, probably other services are checking more tokens and that's why it was the only service failing for me.

Thanks again for your help.
And thank you all for this wonderful tool.
